rubocop-salemove.gemspec

```diff
@@ -1,15 +1,17 @@
-Gem::Specification.new do |s|
+# frozen_string_literal: true
+
+Gem::Specification.new do |s| # rubocop:disable Gemspec/RequireMFA
   s.name        = 'rubocop-salemove'
-  s.version     = '1.2.0'
-  s.date        = '2017-05-04'
+  s.version     = '1.3.0'
   s.summary     = 'RuboCop SaleMove'
   s.description = 'Shared RuboCop configuration for SaleMove projects'
   s.authors     = ['SaleMove']
   s.email       = 'techmovers@salemove.com'
   s.files       = Dir['README.md', 'LICENSE', 'config/*.yml']
   s.homepage    = 'https://github.com/salemove/rubocop-salemove'
   s.license     = 'MIT'
+  s.required_ruby_version = '>= 2.7.0'
 
-  s.add_dependency 'rubocop', '~> 1.36'
-  s.add_dependency 'rubocop-rspec', '~> 2.13'
+  s.add_dependency 'rubocop', '~> 1.65', '>=1.65.1'
```

Comment:
Minimum requirements are there to facilitate features of those requirements this library uses. I see no meaningful changes in the code, meaning we don't actually use any new features.

To me it seems the correct version restrictions would be 

```ruby
s.add_dependency 'rubocop', '~> 1.36' # unchanged, already allows '>=1.65.1'
s.add_dependency 'rubocop-rspec', ['>= 2.13', '< 4.0'] # relaxing requirement from < 3.0 to < 4.0
```

If the aim is to force everyone to use a newer version of rubocop, then this won't do that, it will just stop them upgrading this gem.

If you want to allow higher versions, then loosening upper limits on requirements is fine. If you want to encourage/force newer versions, you should go around the applications and update it. Requiring higher version here for no other benefit is unlikely to help.

Reply:
You are correct. By yesterday evening my thinking was obviously not the best. I will make the changes. Actually the only real problem I had in a consuming repository was with `rubocop-rspec`. Looking at it now, going for `rubocop` itself was not necessary at all. Thank you!

Reply:
Changed it.